website/content/en/highlights/2022-05-18-vrl-iteration-support.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "VRL iteration support has arrived!"
+short: "VRL iteration"
+description: "Remap all your collections using iteration functions."
+authors: ["JeanMertz"]
+date: "2022-05-18"
+pr_numbers: [12317]
+release: "0.22.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+  domains: ["vrl", "remap"]
+tags: ["vector remap language", "vrl", "dsl", "iteration", "enumeration"]
+---
+
+Since we launched **Vector Remap Language** (VRL) [more than a year ago][], the
+language has seen a massive growth in both usage, and feature richness. While we
+intentionally left the core language in a stable state since its launch, we
+_have_ been steadily expanding the [standard library][], increasing
+[performance][], adding [event enrichment features][], and [squashing many
+bugs][].
+
+One of the last major pieces missing from the language — and one requested by
+many of you — is support for mapping collections. We are happy to announce that
+as of Vector 0.22.0, this feature is now available.
```

Comment:
```suggestion
One of the last major pieces missing from the language — and one requested by
many of you — is support for mapping collections. We needed to introduce this
feature thoughtfully since mapping can be a common source of performance
and reliability bugs (i.e., infinite loops). And our design accomplishes this by
providing opinionated mapping functions that align closely with the use case.
So, without further ado, we're happy to announce that as of Vector 0.22.0,
mapping for collections is now available!
```

I took a stab at explaining the reasons and advantages of our approach. Feel free to take it or leave it. I think it's good high level context for the user that helps to continually position Vector as the fast and reliable tool.

Reply:
I like it 👍 Applied in [2bd54ad](https://github.com/vectordotdev/vector/pull/12795/commits/2bd54ad9eecfbe1fe189f0692657aa824b703a31), with a few additional tweaks.